components/src/core/user-menu/user-menu.tsx

```diff
@@ -86,6 +96,7 @@ export const UserMenu: React.FC<UserMenuProps> = (props) => {
                             iconColor={iconColor || menuItem.iconColor}
                             fontColor={fontColor || menuItem.fontColor}
                             backgroundColor={backgroundColor || menuItem.backgroundColor}
+                            styles={{ title: { fontSize: 16, fontWeight: '400' } }}
```

Comment:
I believe specifying the numerical fontWeight doesn't work as expected on one of the OSs. Safer way to do this is to pull the font family/weight from the theme (https://github.com/pxblue/react-native-component-library/blob/dev/components/src/core/list-item-tag/list-item-tag.tsx#L44):
```suggestion
                            styles={{ title: { 
                                fontSize: 16, 
                                fontFamily: theme.fonts.regular.fontFamily,
                                fontWeight: theme.fonts.regular.fontFamily
                            }}}
```